output/transform/postgres.go

```diff
@@ -93,6 +94,18 @@ func transformPostgresDatabases(s snapshot.FullSnapshot, transientState state.Tr
 		}
 
 		s.DatabaseInformations = append(s.DatabaseInformations, &info)
+
+		diff, exist := diffState.DatabaseStats[database.Oid]
```

Comment:
It might be slightly clearer to understand the ages below not actually being a diff if we call this variable `stats` (instead of diff).